lib/client/client.go

```diff
@@ -475,8 +475,13 @@ func (c *NodeClient) handleGlobalRequests(ctx context.Context, requestCh <-chan
 					continue
 				}
 			default:
-				// This handles keepalive messages and matches the behaviour of OpenSSH.
-				r.Reply(false, nil)
+				// This handles keep-alive messages and matches the behaviour of OpenSSH.
+				err := r.Reply(false, nil)
+				if err != nil {
+					log.Warnf("Unable to reply to %v request.", r.Type)
+					continue
+				}
+				log.Debugf("Replied to %v request.", r.Type)
```

Comment:
this line will create a lot of noise, let's remove it. It's ok to warn on failures still though.